src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsEmbeddedFractures.cpp

```diff
@@ -560,6 +612,38 @@ void SolidMechanicsEmbeddedFractures::applySystemSolution( DofManager const & do
                                          domain.getNeighbors(),
                                          true );
 
+  updateState( domain );
+
+}
+
+void SolidMechanicsEmbeddedFractures::updateState( DomainPartition & domain )
+{
+
+  MeshLevel * const meshLevel = domain.getMeshBody( 0 )->getMeshLevel( 0 );
+  ElementRegionManager * const elemManager = meshLevel->getElemManager();
+
+  ConstitutiveManager const * const constitutiveManager = domain.getConstitutiveManager();
+  ContactRelationBase const * const
+  contactRelation = constitutiveManager->getGroup< ContactRelationBase >( m_contactRelationName );
+
+  elemManager->forElementSubRegions< EmbeddedSurfaceSubRegion >( [&]( EmbeddedSurfaceSubRegion & subRegion )
+  {
```

Comment:
@rrsettgast  I think during the dev meeting you and @corbett5 pointed out something to be fixed here.